docs/en/security-privacy-considerations.rst

```diff
@@ -0,0 +1,495 @@
+
+The Wallet validates the OpenID Federation Trust Chain related to the Verifier and its information is displayed to the User before the presentation.
+
+Privacy Requirements
+--------------------
+
+PR-CF-30
+~~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon|
+     - The Credential Format must ensure that there is a robust mechanism to ensure that data that is not to be released to a Verifier cannot be extracted by the Verifier (selective disclosure).
+
+Both SD-JWT and mDOC-CBOR provide selective disclosure capability, allowing Holders to reveal only specific fields to the Verifier. 
+
+PR-CF-40
+~~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |partially-check-icon|
+     - The Credential Format must support correlation protection.
+
+While selective disclosure is a strong tool for preventing correlation, full unlinkability is not guaranteed in all cases. Issues like Verifier collusion or Issuer tracking can arise, especially if sensitive identifiers (e.g., taxpayer ID) are disclosed. 
+
+.. tip::
+  Batch issuance, using different key binding keys and salts for each Credential, can mitigate Verifier/Verifier and presentation unlinkability risks. 
+
+PR-E-60
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon| 
+     - The Trust Framework must ensure that the Issuer cannot learn where the User uses the Credential.
+
+When a Verifier performs the Trust Evaluation of the Issuer of a Credential following [`OID-FED`_], the Issuer cannot know who is the User presenting the Credential.
+In addition, privacy is protected also during the check of the Credential's status. By using Status Assertion [`OAUTH-STATUS-ASSERTION`_], the IT Wallet specification ensures 
+that while the Verifier checks the Credential's validity, the Issuer does not learn where or when the Credential is being used.
+
+PR-E-70
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |partially-check-icon|
+     - The Trust Framework must support correlation protection.
+
+In the case of IT Wallet, as the Trust Framework is based on OpenID Federation [`OID-FED`_], the IT Wallet specification has the following mechanisms in place that could help to reduce the correlation:
+
+- *Verifier-Verifier*: OpenID federation provides a way to check whether the Verifier is asking from the Wallet the information that is authorized to ask or not.  Thus, it will minimize the data exchange and consequently avoid User profiling by colluding between two Verifiers;
+- *Issuer-Verifier*: The Issuer does not require the authentication of the Verifier during the trust evaluation. In principle, the Issuer does not know which Verifiers the User is accessing and will avoid User activity profiling based on the Verifier's access.
+
+.. note::
+ Even if metadata policies and Trust Marks are features offered by OpenID Federation, the current version of the IT Wallet specification makes them optional and does not provide details on their use.
+
```

Comment:
```suggestion
```

Policies using OpenID Federation

Even if metadata policies and Trust Marks are features offered by OpenID Federation, the current version of the IT Wallet specification makes them optional and does not provide details on their use.

Reply:
I don't understand why you are suggesting to remove this text. IMHO, this text sounds like this: if we enable Metadata Policy feature we will reduce the risk of RPs overasking. This, in turn, implies that correlation would be reduced.

Reply:
this note must be removed because it doesn't seem appropriate for the scope of the item

the issuer must not and doesn't actually know the verifier which the credential is about to be used.

policies and trust marks doesn't give nothing in this.

Reply:
With the note we would like to highlight that the  IT-Wallet spec only partially satisfies this requirement. Indeed, the features to have Verifier-Verifier correlation protection (e.g., metadata policies and Trust Marks) are currently optional and not detailed in the spec. Are we wrong?

We can rephrase the note as:

> The requirement is partially satisfied as the features to have Verifier-Verifier correlation protection (e.g., metadata policies and Trust Marks)  are currently optional and not detailed in the IT-Wallet specification.